src/components/ArticleCommentSection.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import { commentSection } from "../utils/api"
+
+
+export default function ArticleCommentSection({article_id}) {
+    const [isError, setIsError]= useState(false)
+    const [isLoading, setIsLoading]= useState(true)
+    const [commentSectionData, setCommentSectionData]= useState()
+
+    console.log(article_id)
+
+    useEffect(()=>{
+        setIsError(false)
+        setIsLoading(true)
+
+        commentSection(article_id)
+        .then((response)=>{
+            console.log(response.comments)
+            setCommentSectionData(response.comments)
+            setIsLoading(false)
+        })
+        .catch((error)=>{
+            setIsError(true)
+        })
+    },[])
+
+    if(isError) return <h1>Error</h1>
+    if(isLoading)return <h1>Loading...</h1>
+
+  return (
+    <div className="commentSection">
+        {commentSectionData.map((comment)=>{
+            return (
+            <div className="comment" key={comment.comment_id}>
```

Comment:
This could be it's own component that's only job is to render the comment that can be passed down in props